hadoop-hdds/docs/content/interface/SwaggerReconApi.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Interactive Recon API Specification
+type: custompage
+layout: swagger-page
+---
+
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+{{< swagger-ui url="/swagger-resources/recon-api.yaml" >}}
```

Comment:
We have set this path as a hardcoded value, this can change from env to env right?
For example in my docker setup we have this file placed at the following path :- 
```
/docs/swagger-resources/recon-api.yaml 
```
We need to set it dynamically according to the prod env.

Reply:
Hello @ArafatKhan2198 , thanks for asking the question. While it is true that we would need to handle this kind of scenario based on the prod env, I think for docs we do not build it as a part of the overall Ozone parcel, but rather in the ozone-site repo via GitHub actions.

After testing it out, it seems for the hosted site, the relative path would be:
`/docs/egde/swagger-resources/recon-api.yaml`

<img width="1728" alt="Screenshot 2023-11-06 at 20 30 28" src="https://github.com/apache/ozone/assets/43001336/acdf5997-f19c-4832-a43b-df9c393cd14c">

How I tested it is:
* Ran the following in the Ozone main repo:
```
mvn --no-transfer-progress -pl :hdds-docs clean package
```
* Built the master branch for `ozone-site` repo using `hugo`
* In the public folder that was generated, I created a folder named **edge**
* Copied the files from `hadoop-hdds/docs/target/classes/docs` in the Ozone repo to the above public folder.
* Hosted the site locally and tested the location manually.

@adoroszlai @errose28 This would be the correct way that the ozone-site gets generated right? So this location should be good.